### DISCUSSION TITLE
flink-cdc支持表结构同步么？有实现的么？

### DISCUSSION THREAD

**Discussion by AppleJunJiang** [apache/flink-cdc#3781] [Q&A]
flink-cdc支持表结构同步么？有实现的么？

**Comment by beryllw** [comment]
Pipeline connector support [schema-evolution](https://nightlies.apache.org/flink/flink-cdc-docs-release-3.2/docs/core-concept/schema-evolution/).

**Comment by Kryzeno** [comment]
支持 但是容易出问题 整个作业报错 源表数据量越大越容易出问题
